src/components/@shared/atoms/Container/index.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { ReactNode } from 'react'
+import { ComponentStory, ComponentMeta } from '@storybook/react'
+
+import Container from '@shared/atoms/Container'
+import styles from './index.module.css'
+
+export default {
+  title: 'Component/@shared/atoms/Container',
+  component: Container
+} as ComponentMeta<typeof Container>
+
+const Template: ComponentStory<typeof Container> = (args) => (
+  <Container {...args} />
+)
+
+interface Props {
+  args: {
+    narrow: boolean
+    children: ReactNode
+    className: string
+  }
+}
+
+export const Primary: Props = Template.bind({})
+Primary.args = {
+  narrow: true,
+  children: (
```

Comment:
What is going on here, as this is nesting under `Container` a new div which then has the same styles again as the `Container` we are already testing? `children` should just be the actual content we put in the `Container`, so something like this:

```js
children:  <>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris aliquam facilisis molestie. Integer eget congue turpis, in pharetra lectus. Sed urna dolor, porttitor luctus mauris eget, lacinia consectetur eros. Duis consequat, turpis et porttitor cursus, ante lacus placerat arcu, vel pellentesque enim orci ac sem.</>
```